pkg/asset/templates/node-agent.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: node-agent
+  namespace: kube-system
+  labels:
+    k8s-app: node-agent
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: node-agent
+    spec:
+      containers:
+      - name: node-agent
+        image: quay.io/derek_parker/node-agent:f2e3b7224e7a048971a33f1df36c2d86812dcc73
```

Comment:
Before merging we might want to break this into 2 steps. Merge the tool & build the image, then merge this manifest with an official image.